doc/topics/grains/index.rst

```diff
@@ -95,44 +95,17 @@ the following configuration:
 
 .. code-block:: yaml
 
-    'node_type:webserver':
+    'roles:webserver':
       - match: grain
-      - webserver
-
-    'node_type:postgres':
-      - match: grain
-      - postgres
+      - state0
 
-    'node_type:redis':
+    'roles:memcache':
       - match: grain
-      - redis
-
-    'node_type:lb':
-      - match: grain
-      - lb
+      - state1
+      - state2
 
 For this example to work, you would need to have defined the grain
-``node_type`` for the minions you wish to match. This simple example is nice,
-but too much of the code is similar. To go one step further, Jinja templating
```

Comment:
Consistency is good but wouldn't it also be good to leave in an example of Jinja templating?